python/ambassador/config/resourcefetcher.py

```diff
@@ -308,25 +302,16 @@ def handle_k8s(self, raw_obj: dict) -> None:
                 # Nothing else to do.
                 return
 
-        handler = None
-        check_dup = True
-
-        if obj.gvk.kind in CRDTypes:
-            handler = self.handle_k8s_crd
-            # self.handle_k8s_crd will do its own dup checking.
-            check_dup = False
-        else:
-            handler_name = f'handle_k8s_{obj.gvk.kind.lower()}'
-            # self.logger.debug(f"looking for handler {handler_name} for K8s {kind} {name}")
-            handler = getattr(self, handler_name, None)
+        handler_name = f'handle_k8s_{obj.gvk.kind.lower()}'
```

Comment:
I kinda feel like there should be convenience accessors for `obj.gvk.kind` and `obj.gvk.kind.lower()`. 🙂  Probably a separate PR...

Reply:
This is actually a super small change so I'll throw it in here as well. Although there are actually almost no places where the `.lower()` is still used (other than generating the domain in the GVK itself), so maybe just `.kind`?